plugins/techdocs/api-report.md

```diff
@@ -95,6 +95,7 @@ export type DocsTableProps = {
   entities: Entity[] | undefined;
   title?: string | undefined;
   loading?: boolean | undefined;
+  options?: object | undefined;
```

Comment:
Could you create a specific `TableOptions` type to help people understand what properties can be set on this object?

Example:
```ts
type TableOptions = {
    /**  Describe property here */
    paging: boolean,
   /**  Describe property here */
    pageSize: number,
    /**  Describe property here */
    search: boolean,
    /**  Describe property here */
    actionsColumnIndex: number,
  };
  
  export type DocsTableProps = {
  entities: Entity[] | undefined;
  title?: string | undefined;
  loading?: boolean | undefined;
  options?: object | undefined;
Member
 camilaibs Pending

Could you create a specific TableOptions type to help people understand what properties can be set on this object?

Example:

type TableOptions = {
    /**  Describe property here */
    paging: boolean,
   /**  Describe property here */
    pageSize: number,
    /**  Describe property here */
    search: boolean,
    /**  Describe property here */
    actionsColumnIndex: number,
  };

export type DocsTableProps = {
  entities: Entity[] | undefined;
  title?: string | undefined;
  loading?: boolean | undefined;
  options?: TableOptions | undefined;
  columns?: TableColumn<DocsTableRow>[];
  actions?: TableProps<DocsTableRow>['actions'];
};
```

Reply:
thanks for the response, updated now to reflect `Options` from `@material-table/core`